packages/bpk-component-banner-alert/stories.js

```diff
@@ -47,10 +47,40 @@ const BpkBannerAlertExpandableState = withBannerAlertState(
 
 storiesOf('bpk-component-banner-alert', module)
   .add('Neutral', () => (
-    <BpkBannerAlert message={message} type={ALERT_TYPES.NEUTRAL} />
+    <div>
```

Comment:
Added group examples to each story to show what they look like in each variant and during testing of code updates.

Will resolve the styling later once we have full colour sign-off